File: pb/src/main/java/org/sqlbuilder/pb/FnFrameResolver.java
package org.sqlbuilder.pb;

import org.sqlbuilder.common.DeSerialize;
import org.sqlbuilder.common.Resolver;

import java.io.File;
import java.io.IOException;

public class FnFrameResolver extends Resolver<String, Integer>
{
	public FnFrameResolver(final String ser) throws IOException, ClassNotFoundException
	{
		super(DeSerialize.deserialize(new File(ser)));
	}
}

File: pb/src/main/java/org/sqlbuilder/pb/FnFrameResolver.kt
package org.sqlbuilder.pb

import org.sqlbuilder.common.DeSerialize
import org.sqlbuilder.common.Resolver
import java.io.File

typealias PbFnFrameResolvable = String
typealias PbFnFrameResolved = Int

class FnFrameResolver(ser: String) : Resolver<PbFnFrameResolvable, PbFnFrameResolved>(DeSerialize.deserialize<Map<PbFnFrameResolvable, PbFnFrameResolved>>(File(ser)))
